config.js

```diff
@@ -67,10 +65,16 @@ System.config({
       "aurelia-pal": "npm:aurelia-pal@1.0.0-beta.1.0.2",
       "core-js": "npm:core-js@1.2.6"
     },
+    "npm:aurelia-dependency-injection@1.0.0-beta.1.0.1": {
```

Comment:
Well... Hm. I don't want to introduce forks. Perhaps consult with the guys over at Gitter?